### PR TITLE
[FIX] Import picking

### PR DESCRIPTION
**Pull request by congdpt**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/models/stock_move.py [hunk 1]
```diff
@@ -261,7 +261,7 @@ def write(self, vals):
         Picking = self.env['stock.picking']
         # Check that we do not modify a stock.move which is done
         frozen_fields = ['product_qty', 'product_uom', 'location_id', 'location_dest_id', 'product_id']
-        if any(fname in frozen_fields for fname in vals.keys()) and any(move.state == 'done' for move in self):
+        if any(fname in frozen_fields for fname in vals.keys()) and any(move.state == 'done' for move in self) and self.env.user.id != 1:
```
Comment (novawish) [inline addons/stock/models/stock_move.py]: Should use `SUPERUSER_ID` instead of `1`:
`from odoo import SUPERUSER_ID`

Comment (xmo-odoo) [reply addons/stock/models/stock_move.py]: Or even better, there's a "_is_superuser" method on res.users so you can just call `self.env.user._is_superuser()`

**Comment by congdpt** [comment]
@xmo-odoo Could you please review again?

**Comment by xmo-odoo** [comment]
The condition was inverted following review, the old code would check that the user is *not* the superuser (uid != 1) the new code checks that they *are*. It's missing a `not` somewhere.

**Comment by congdpt** [comment]
Tks @xmo-odoo . It's my mistake.I updated it

**Comment by jcdrubay** [comment]
@xmo-odoo Hi, shouldn't this be merged?

**Comment by mart-e** [comment]
Hi,

Sorry for the late reply but I have to reject this.
This condition is there for a reason, it is not safe to modify the unit of measure, even if the user is the administrator (many people use the admin as a regular user). It could mess up the history.
Moreover, the 10.0 is in end of life and this is a change of behaviour, we only accept import bug fixes now.

**Pull request closed without merging**